gpkitmodels/Commercial Sizing/VT_simple_profile.py

```diff
@@ -327,6 +327,7 @@ def __init__(self, alt, **kwargs):
         subs = None
 
         Model.__init__(self, None, constraints, subs)
+
 class Engine(Model):
     """
     place holder engine model
```

Comment:
@bjarni177 look at this file and perform a similar integration with your fuselage